common/src/main/java/com/paypal/test/utilities/logging/SimpleLogger.java

```diff
@@ -432,7 +432,7 @@ public void log(Level level, String msg, Object param) {
     }
 
     @Override
-    public void log(Level level, String msg, Object... params) {
+    public void log(Level level, String msg, Object[] params) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1312203139/issues/source?bid=2928425&fileBranchId=2952589#l435)